frame/system/src/extensions/check_nonce.rs

```diff
@@ -107,7 +107,7 @@ impl<T: Trait> SignedExtension for CheckNonce<T> where
 		};
 
 		Ok(ValidTransaction {
-			priority: info.weight as TransactionPriority,
```

Comment:
Don't know what our take on `SignedExtensions` composability is, but we should make it clear that `CheckWeight` is now required, cause without it the prioritisation would not work at all.

Reply:
is it an issue if there is no prioritisation ?
I agree we should make clear that chain should handle their prioritisation correctly for production chain, but a dev chain can work without it no ?

Reply:
Ahh so this must have been the reason why we set the weight as priority in `CheckWeight` and `CheckNonce`. 

I am ambivalent. Honestly, at the end of the day, I think all 3 of these signed extensions are absolutely vital for any chain: CheckNonce, CheckWeight, ChargeFee, and if someone is to faff around with them they should be super careful about the detail, so I am still in for not setting any priority here. I will also remove it from `CheckWeight`.

Reply:
Actually, no, `CheckWeight` is important because it is the only one that is bumping `Operational` class.

Reply:
And I think this is correct because setting the distinction of Normal and Operational should be the concern of the system module, not TransactionPayment.